src/Service.GraphQLBuilder/GraphQLUtils.cs

```diff
@@ -46,6 +46,7 @@ public static bool IsBuiltInType(ITypeNode typeNode)
             {
                 "ID",
                 BYTE_TYPE,
+                GUID_TYPE,
```

Comment:
How is "ID" different than GUID_TYPE?

Reply:
Actually, this change I think is not required. GUID_TYPE is not a built-in type for hotchocolate. Guids correspond to ID type in Hotchocolate : https://chillicream.com/docs/hotchocolate/v13/defining-a-schema/scalars#id.